snowflake/datadog_checks/snowflake/queries.py

```diff
@@ -52,7 +52,7 @@
 
 # https://docs.snowflake.com/en/sql-reference/account-usage/warehouse_metering_history.html
 WarehouseCreditUsage = {
-    'name': 'billings.warehouse.metrics',
+    'name': 'billing.warehouse.metrics',
```

Comment:
this was wrong?

Reply:
It was not incorrect, just inconsistent with other naming conventions. The name is not used anywhere besides logging. I can make this change in a separate PR if you think it's cleaner.